src/frontend/planner_test/tests/testdata/output/nexmark_source.yaml

```diff
@@ -2209,7 +2209,7 @@
     StreamRowIdGen { row_id_index: 7 }
     └── StreamSource { source: bid, columns: [auction, bidder, price, channel, url, date_time, extra, _row_id] } { source state table: 8 }
 
-    Table 0 { columns: [ id, item_name, count(auction), $expr1 ], primary key: [ $2 DESC, $0 ASC, $1 ASC, $3 ASC ], value indices: [ 0, 1, 2, 3 ], distribution key: [], read pk prefix len hint: 0 }
+    Table 0 { columns: [ id, item_name, count(auction), $expr1 ], primary key: [ $2 DESC, $0 ASC, $1 ASC ], value indices: [ 0, 1, 2, 3 ], distribution key: [], read pk prefix len hint: 0 }
```

Comment:
Explanation: table 0 is global topn

Previously, we deleted vnode in global topn's stream key, but when `infer_internal_table_catalog`, it still uses input stream key, which contains vnode. Now we removed vnode from input stream key, so this is changed.